package.json

```diff
@@ -11,7 +11,7 @@
     "dev:watch": "nodemon -r ts-node src/index.ts",
     "test": "jest --coverage",
```

Comment:
I think you can add NODE_ENV= test here so as to be used in the sequelize config.

Reply:
Worked on the feedback.